### ISSUE TITLE
Feature Idea: Most used client

### ISSUE THREAD

**Issue by 1hitsong** [Chaphasilor/jellyfin-rewind#21]
How are users listening to their music? Jellyfin web client? Finamp? Feishin? viv?

**Comment by Chaphasilor** [comment]
Yeah, that's been on my list for a long time. A pie chart would also be nice here.  
We could additionally differentiate by platform (mobile vs. desktop vs. misc)?
